webapp/src/main/java/org/apache/atlas/web/rest/TaskREST.java

```diff
@@ -70,4 +71,23 @@ public TaskSearchResult getTasks(TaskSearchParams parameters) throws AtlasBaseEx
             AtlasPerfTracer.log(perf);
         }
     }
+
+    @PUT
+    @Path("retry/{guid}")
```

Comment:
Can there be a case to submit multiple tasks to retry?
Rare case, but just thinking of having support for multiple GUIDs 
@DillionTan , please help to decide

Reply:
think we don't need it now, anyway if we want multiple we can still add on later

Reply:
Noted